src/test/java/io/vertx/core/net/SocketAddressTest.java

```diff
@@ -14,80 +15,80 @@ public class SocketAddressTest extends VertxTestBase {
 
   @Test
   public void testInetSocketAddressFromUnresolvedAddress() {
-    SocketAddress addr = SocketAddress.inetSocketAddress(InetSocketAddress.createUnresolved("localhost", 8080));
+    SocketAddress addr = SocketAddress.inetSocketAddress(InetSocketAddress.createUnresolved("localhost", HttpTestBase.DEFAULT_HTTP_PORT));
```

Comment:
those tests don't use a server, so they should use literal values instead, that makes them less readable